script/openqa-bootstrap-container

```diff
@@ -79,6 +84,6 @@ systemctl start systemd-nspawn-openqa@$CONTAINER_NAME
 # ensure that the container is really running
 # ignore expected errors about 'Failed to create bus connection: Protocol error' and restarting error
 while ! timeout -s9 2 systemd-run -qPM $CONTAINER_NAME /bin/bash -c whoami /dev/null 2>&1 ; do systemctl restart systemd-nspawn-openqa@$CONTAINER_NAME.service || true ; sleep 3 ; done
-systemd-run -qPM $CONTAINER_NAME /bin/bash -c '/usr/share/openqa/script/openqa-bootstrap'
+systemd-run "${systemd_run_params[@]}" /bin/bash -c '/usr/share/openqa/script/openqa-bootstrap'
```

Comment:
Was this actually tested? Because this seems to drop the `-q` and the `-M $CONTAINER_NAME` parameters and I don't see how this could work without them.
Also in the line above `while ! timeout -s9 2 systemd-run -qPM $CONTAINER_NAME …` the `-P` option is used as well.

Reply:
sorry - somehow overlooked the line that set that params.